src/components/DisplayArticlesData.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import { getArticleData } from "../modules/articlesData";
+import { Card, Image } from "semantic-ui-react";
+
+const DisplayArticlesData = () => {
+  const [articlesData, setArticlesData] = useState([]);
+  const getArticlesData = async () => {
+    let result = await getArticleData();
```

Comment:
to me these two names are a bit too similar... Think about the readability

Reply:
Yep. Got this. We will modify that. Thanks.